Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -1053,11 +1063,13 @@ theorem integrable_mul_const_iff {c : 𝕜} (hc : IsUnit c) (f : α → 𝕜) :
 
 theorem Integrable.bdd_mul' {f g : α → 𝕜} {c : ℝ} (hg : Integrable g μ)
     (hf : AEStronglyMeasurable f μ) (hf_bound : ∀ᵐ x ∂μ, ‖f x‖ ≤ c) :
```

Comment:
What do you think about deprecating the current [MeasureTheory.Integrable.bdd_mul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/L1Space/Integrable.html#MeasureTheory.Integrable.bdd_mul) and calling this `bdd_mul` instead? Otherwise we would have to introduce similar duplication for everything you're doing...